packaging/linux/provision.sh

```diff
@@ -7,27 +7,47 @@ AWS_ACCESS_KEY=$4
 CLI_BRANCH=$5
 PACKAGER_BRANCH=$6
 export REPO=$7
-export CORE_TAG_NAME="4.2.dev1"
+export CORE_TAG_NAME="4.1"
 export CORE_BRANCH="master"
 
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-packager/add-osx-support/common/provision.sh -o ./common-provision.sh &&
+    source common-provision.sh
+else
+    curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-packager/${CORE_BRANCH}/common/provision.sh -o ./common-provision.sh &&
+    source common-provision.sh
+fi
+
 curl -u $GITHUB_USERNAME:$GITHUB_PASSWORD https://raw.githubusercontent.com/cloudify-cosmo/${REPO}/${CORE_BRANCH}/packages-urls/common_build_env.sh -o ./common_build_env.sh &&
 source common_build_env.sh &&
-curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-packager/${CORE_BRANCH}/common/provision.sh -o ./common-provision.sh &&
-source common-provision.sh
 curl -u $GITHUB_USERNAME:$GITHUB_PASSWORD https://raw.githubusercontent.com/cloudify-cosmo/${REPO}/${CORE_BRANCH}/packages-urls/manager-single-tar.yaml -o ./manager-single-tar.yaml &&
 export SINGLE_TAR_URL=$(cat manager-single-tar.yaml)
 
 
 install_common_prereqs &&
 rm -rf cloudify-cli
-git clone https://github.com/cloudify-cosmo/cloudify-cli.git
-cd cloudify-cli/packaging/omnibus
-gitTagExists=$(git tag -l $CORE_TAG_NAME)
-if [ "$CORE_BRANCH" != "master" ]; then
-    git checkout -b ${CORE_BRANCH} origin/${CORE_BRANCH}
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    git clone -b add-osx-support https://github.com/cloudify-cosmo/cloudify-cli.git
 else
-    git checkout ${CORE_BRANCH}
+    git clone https://github.com/cloudify-cosmo/cloudify-cli.git
 fi
+cd cloudify-cli/packaging/omnibus
+#gitTagExists=$(git tag -l $CORE_TAG_NAME)
+#if [ "$CORE_BRANCH" != "master" ]; then
+#    git checkout -b ${CORE_BRANCH} origin/${CORE_BRANCH}
+#else
+#    git checkout ${CORE_BRANCH}
+#fi
```

Comment:
Can we please avoid leaving commented code lying around? We can always recover it through git if needed, so leaving it in and commented is just providing meaningless filler for anyone maintaining this later.